src/shared/components/challenge-detail/Header/TabSelector/index.jsx

```diff
@@ -61,8 +62,7 @@ export default function ChallengeViewSelector(props) {
           <a
             onClick={(e) => { e.preventDefault(); onSelectorClicked('SUBMISSIONS'); }}
             styleName={getSelectorStyle(selectedView, 'SUBMISSIONS', trackLower)}
-          >
-            SUBMISSIONS
+          >SUBMISSIONS {numSubmissions ? `(${numSubmissions})` : ''}
```

Comment:
No need to make a check here, I believe: you have already specified in `ChallengeViewSelector.defaultProps` that default for `numSubmissions` is zero. So, even if no value is passed by parent component via props, it will be automatically set to 0 anyway, so you can just render `{numSubmissions}`. Please, change, and double-check that it works fine.